### DISCUSSION TITLE
About Current Pull Requests

### DISCUSSION THREAD

**Discussion by hasanmanzak** [Biarity/Sieve#131] [General]
As of now (14.05.2021 23:25 UTC) there are 8 open pull requests:

- [x] ~~#65~~ 
- [x] ~~#79~~ 
- [x] ~~#109~~
- [x] #112 
- [x] ~~#113~~
- [x] #114 
- [ ] #117 &
- [ ] #125 

@ITDancer13, @a-patel and I are writing back to the authors to resolve the current conflicts. Some PRs are very old, some are very new but the problem remains: there are conflicts and we are expecting people to fix them.

I say lets checkout the PRs ourselves and resolve the conflicts, if any, and merge the fixes/changes and close those long waited issues.. That way we can start a clean page and say, do an enforcement about contribution like "write the unit tests about the fix/feature you are contributing", or close the PR if builds are failing or something or some other thing I couldn't thought about.

Any thoughts?

**Comment by ITDancer13** [comment]
Another great idea :-)
#113 is ready to be merged

**Reply by hasanmanzak** [reply]
Merged

**Comment by hasanmanzak** [comment]
Closed #65. It was superseeded by #117 
Closed #79. It was superseeded by #125

**Reply by hasanmanzak** [reply]
#109 is closed but merged the functionality with #134

**Comment by ITDancer13** [comment]
I'll take #112 now. Resolve the conflicts and check the result.
